games-roguelike/angband/angband-4.1.3.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools eapi7-ver eutils gnome2-utils
+
+MAJOR_PV=$(ver_cut 1-2)
+
+DESCRIPTION="A roguelike dungeon exploration game based on the books of J.R.R. Tolkien"
+HOMEPAGE="https://rephial.org/"
+SRC_URI="https://rephial.org/downloads/${MAJOR_PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="ncurses sdl +sound X"
+
+RDEPEND="X? ( x11-libs/libX11 )
+	!ncurses? ( !X? ( !sdl? ( sys-libs/ncurses:0=[unicode] ) ) )
+	ncurses? ( sys-libs/ncurses:0=[unicode] )
+	sdl? ( media-libs/libsdl[video,X]
+		media-libs/sdl-image
+		media-libs/sdl-ttf
+		sound? ( media-libs/libsdl[sound] )
```

Comment:
Would something like `X -sdl sound` still produce working sound? What about ncurses? 
(I am looking at if sound always requires sdl to be set, too, or if this dep is just needed with sdl - then this is correct)

Reply:
Yes, sound requires sdl, Angband uses SDL for sound support.

Reply:
So if I do `X ncurses sound -sdl`, it won't give me sound even though I believe that to happen? You really need to add REQUIRED_USE to satisfy that condition.

Reply:
Indeed. You are right. I'll add `REQUIRED_USE` then.